pytket/phir/phirgen.py

```diff
@@ -220,7 +220,18 @@ def convert_subcmd(op: tk.Op, cmd: tk.Command) -> JsonDict | None:
     out: JsonDict | None = None
     match op:  # non-quantum op
         case tk.BarrierOp():
-            out = {"meta": "barrier", "args": [arg_to_bit(qbit) for qbit in cmd.qubits]}
+            if "sleep" in op.data:
+                dur = op.data.removeprefix("sleep(").removesuffix(")")
+                out = {
+                    "mop": "Idle",
+                    "args": [arg_to_bit(qbit) for qbit in cmd.qubits],
+                    "duration": (dur, "s"),
+                }
+            else:
```

Comment:
There are a bunch of other keywords that can appear in `data`: https://github.com/CQCL/tket/blob/d854c8971934f6e5ee153de94a1d3e9a2c29a143/pytket/pytket/qasm/qasm.py#L419 . I wonder if, rather than ignore the others, it would be better to:
- first check if `data` is empty, treat as a normal barrier if so;
- then check if it is a `sleep` command, and convert it;
- otherwise, raise an exception explaining that we don't support that in the conversion (yet).

Reply:
Or maybe it is fine to just ignore the "order" and "group" commands? Up to you.

Reply:
> There are a bunch of other keywords that can appear in `data`: [CQCL/tket@`d854c89`/pytket/pytket/qasm/qasm.py#L419](https://github.com/CQCL/tket/blob/d854c8971934f6e5ee153de94a1d3e9a2c29a143/pytket/pytket/qasm/qasm.py#L419) . I wonder if, rather than ignore the others, it would be better to:
> 
> * first check if `data` is empty, treat as a normal barrier if so;
> * then check if it is a `sleep` command, and convert it;
> * otherwise, raise an exception explaining that we don't support that in the conversion (yet).

Great point, let me incorporate some of these suggestions!